### DISCUSSION TITLE
21-06-09 일일회고

### DISCUSSION THREAD

**Discussion by GleamingStar** [d-h-k/issue-tracker#26] [회고]
좀 많이 늦게 올렸습니다! 
ㅎㅎ...ㅈㅅ!

**Comment by GleamingStar** [comment]
바닐라JS로 했을 땐 쉬웠던 일괄 선택 / 해제 기능이 리액트에서 하니 너무 어려웠다
차라리 recoil 때리고 했으면 쉬웠을 것 같은데
내일은 미루고 미루던 recoil적용 🎪

**Comment by eeeesong** [comment]
잭슨과 함께 탭바 부분 UI를 만들고 리팩토링까지 진행했습니다!
애플 로그인 기능 진행 중인데 아직은 아리송하네여
오늘도 다들 고생하셨습니다~

**Comment by Jiwon-JJW** [comment]
서버 배포를 시작했습니다!
반자동화도 진행해서, 앞으로 슬랙에 배포봇이 올라올 예정입니다~!
내일은 iOS oauth 로그인 구현을 완료 할 수 있지 않을까! 합니다~

**Comment by reminderclock** [comment]
아주 간단한 타입스크립트를 적용 시키고 오오스 공부를 좀 했는데 개념이 잘 와닿지가 않네요ㅎㅎ

**Comment by JacksonPk** [comment]
깃헙로그인 관련 체크를 해보았습니다. 오늘 로그인 진행을 잘 해보면 될 것 같아요!
